sa/sa_test.go

```diff
@@ -2574,3 +2578,139 @@ func TestIncidentsForSerial(t *testing.T) {
 	test.AssertNotError(t, err, "Failed to retrieve incidents for serial")
 	test.Assert(t, len(incidentsForSerial) == 1, "No active incidents returned")
 }
+
+type inmemSA struct {
+	sapb.StorageAuthorityClient
+	Impl *SQLStorageAuthority
+}
+
+type sfiData struct {
+	serial *sapb.IncidentSerial
+	err    error
+}
+
+type sfiClient struct {
+	// grpc.ClientStream is embedded to meet the
+	// StorageAuthority_SerialsForIncidentClient interface.
+	grpc.ClientStream
+	sfiChan <-chan sfiData
+}
+
+// Recv receives a single serial from the server.
```

Comment:
Comment should explain what it does once it's received a serial. Also, both `sfiClient.Recv` and `sfiServer.Send` are receiving in one place (channel or parameter) and sending in another place (return value or channel). To document these we should explain both, and their relation to each other.

Recv, Send, and Context are all implemented in order to satisfy some interface. Can you document what that interface is?